generate/generate.go

```diff
@@ -1629,7 +1651,11 @@ func (s *service) generateNewAPICallFunc(a *API) {
 		pn("		time.Sleep(500 * time.Millisecond)")
 		pn("	}")
 	} else {
-		pn("	resp, err := s.cs.newRequest(\"%s\", p.toURLValues())", a.Name)
+		if a.Name == "deployVirtualMachine" || a.Name == "login" || a.Name == "updateVirtualMachine" {
```

Comment:
Just one suggestion, a.Isasync can be used to determine if a Post request be made as for most async APIs, POST can be used.

Reply:
That's a good point. I'm hesitant to default async APIs to use POST just yet, that might need more thorough testing.  Definitely something that can be refined though, rather than sticking to just these few named APIs.

Reply:
Makes sense, we can do that in the next iteration.